scripts/conda_to_pip.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
```

Comment:
maybe change this name to

``generate_requirements-dev_from_environment`` or some more meaningful name